src/backend/main.mo

```diff
@@ -0,0 +1,137 @@
+import Array "mo:base/Array";
+import Debug "mo:base/Debug";
+import Float "mo:base/Float";
+import HashMap "mo:base/HashMap";
+import Int "mo:base/Bool";
+import Iter "mo:base/Iter";
+import Nat "mo:base/Nat";
+import Principal "mo:base/Principal";
+import Time "mo:base/Time";
+import Types "./types";
+
+
+// shared ({ caller = creator }) actor class Metascore (init : ?Types.InitObject) = canister {
+shared ({ caller = creator }) actor class Metascore () = canister {
+
+
+    ////////////
+    // State //
+    //////////
+
+    stable var owners : [Principal] = [];
+    stable var gameCanDir : [Types.GameCanRecord] = [];
+    // I'm sure theres a better data model to use. Tries?
+    stable var playerScores : [(Types.Player, Types.GameCanRecord, Types.Score)] = [];
```

Comment:
I'll let this slide for, since this is a POC....

Reply:
If you are interested in writing the data model that would be great 😆

Reply:
Coming up. 🔜

Reply:
We first have to make the data model of course. 😊